components/src/molecules/collapse/Collapse.scss

```diff
@@ -0,0 +1,150 @@
+$collapse__button-text-decoration     : underline !default;
+$collapse__button-background          : $white !default;
+$collapse__button-background-active   : $gray-darker !default;
+$collapse__button-outline             : none !default;
+$collapse__button_color--inner        : $white !default;
+$collapse__button_color-active--inner : $color-primary !default;
+$collapse__button_background--inner   : $gray-dark !default;
+$collapse__icon-width                 : $spacer--medium !default;
+$collapse__icon-height                : $spacer--medium !default;
+$collapse__icon-fill                  : $black !default;
+$collapse__icon-fill-active           : $color-primary !default;
+$collapse__icon-fill--secondary       : $gray !default;
+$collapse__icon-fill-active--secondary: $color-primary !default;
+$collapse__icon-margin-left--secondary: $spacer--medium !default;
+$collapse__icon-inner                 : $color-primary !default;
+$collapse__content-padding            : $spacer !default;
+
+.a-collapse {
+  position: relative;
+  width: $collapse__width;
+  background-color: $collapse__background;
+
+  &--secondary {
+    .a-collapse__button {
+      color: $collapse__button-color;
+      font-weight: $collapse__button-font-weight;
+      text-transform: $collapse__button-text-transform;
+      background-color: $collapse__button-background;
+      outline: $collapse__button-outline;
+
+      &:hover,
+      &:active,
+      &:focus {
+        color: $collapse__button-color;
+        background-color: $collapse__button-background;
+
+        & > .a-collapse__icon {
+          fill: $collapse__icon-fill--secondary;
+        }
+      }
+
+      &[aria-expanded="true"] {
+        color: $collapse__button-color-active;
+        text-decoration: $collapse__button-text-decoration;
+        background-color: $collapse__button-background;
+
+        &:active,
+        &:focus,
+        &:hover {
+          color: $collapse__button-color-active;
+          background-color: $collapse__button-background;
+
+          & > .a-collapse__icon {
+            fill: $collapse__icon-fill-active--secondary;
+          }
+        }
+      }
+    }
+
+    .a-collapse__icon {
+      position: relative;
+      top: 3px;
+      display: inline;
+      margin-left: $collapse__icon-margin-left--secondary;
+      fill: $collapse__icon-fill--secondary;
+    }
+  }
+
+  &--inner {
+    .a-collapse__button {
+      color: $collapse__button_color--inner;
+      background-color: $collapse__button_background--inner;
+
+      &:active,
+      &:focus {
+        color: $collapse__button_color-active--inner;
+      }
+    }
+
+    .a-collapse__icon {
+      fill: $collapse__icon-inner;
+    }
+  }
+
+  &__button {
+    position: relative;
+    display: block;
+    width: $collapse__button-width;
+    margin: $reset;
+    padding: $collapse__button-padding;
+    color: $black;
+    text-align: left;
+    text-decoration: none;
+    border: 0;
+    cursor: pointer;
+
```

Comment:
Add base transition and please update icon color to green on hover
![Zrzut ekranu 2019-05-15 o 12 45 17](https://user-images.githubusercontent.com/7126345/57770036-acdead00-770f-11e9-8f4d-1b15f8a90a4f.png)